### ISSUE TITLE
Add Basic Recipes

### ISSUE THREAD

**Issue by aod6060** [aod6060/elemental_infusion#1]
This is basically adding in the vanilla recipes for certain metals and objects that won't be apart of the multiblock recipes.

**Comment by aod6060** [comment]
Lets get this started.

**Comment by aod6060** [comment]
OK I'm done with the basic recipes for now. These are vanilla recipes that are crafted in a crafting table. I've taken a few pictures of the current list of recipes that are in the mod.

Crafting Fire Block
![2015-10-19_22 56 15](https://cloud.githubusercontent.com/assets/1819755/10598351/ce4ccefa-76bd-11e5-8d54-4a0cda991c69.png)
Crafting Water Block
![2015-10-19_22 56 23](https://cloud.githubusercontent.com/assets/1819755/10598353/d3b6136a-76bd-11e5-824c-93f1b5a7234e.png)
Crafting Air Block
![2015-10-19_22 56 37](https://cloud.githubusercontent.com/assets/1819755/10598357/d89f9a68-76bd-11e5-8164-e411bb7fba84.png)
Crafting Earth Block
![2015-10-19_22 56 44](https://cloud.githubusercontent.com/assets/1819755/10598361/dd19c906-76bd-11e5-8517-756b1bc6e5c1.png)
Crafting 9 Fire Dust
![2015-10-19_22 56 49](https://cloud.githubusercontent.com/assets/1819755/10598363/e2a950a8-76bd-11e5-80ab-f3503855a8ca.png)
Crafting 9 Water Dust
![2015-10-19_22 56 53](https://cloud.githubusercontent.com/assets/1819755/10598364/e5f162c8-76bd-11e5-85d8-6d21de2a8e42.png)
Crafting 9 Air Dust
![2015-10-19_22 56 57](https://cloud.githubusercontent.com/assets/1819755/10598370/f226383e-76bd-11e5-876b-691fe7aa233a.png)
Crafting 9 Earth Dust
![2015-10-19_22 57 00](https://cloud.githubusercontent.com/assets/1819755/10598376/f7ff9840-76bd-11e5-8583-1ebd8eebd32e.png)
Crafting Void Stone
![2015-10-19_22 59 33](https://cloud.githubusercontent.com/assets/1819755/10598389/0d143dee-76be-11e5-98d1-3346df980075.png)
Note: Totem Elemental Heads might be changed in the 
          Upcoming update. I'm thinking of replacing it with
          Elemental Dust Blocks
Crafting Fire Totem Head
![2015-10-19_23 00 00](https://cloud.githubusercontent.com/assets/1819755/10598412/5d086816-76be-11e5-81ec-538b195264e7.png)
Crafting Water Totem Head
![2015-10-19_23 00 16](https://cloud.githubusercontent.com/assets/1819755/10598413/622aff70-76be-11e5-8440-8cea499dde1e.png)
Crafting Air Totem Head
![2015-10-19_23 00 38](https://cloud.githubusercontent.com/assets/1819755/10598417/680b0c78-76be-11e5-9085-c67cf6a48e63.png)
Crafting Earth Totem Head
![2015-10-19_23 00 49](https://cloud.githubusercontent.com/assets/1819755/10598420/6d71db06-76be-11e5-8283-2ff3f5002ddf.png)
Crafting Void Glass.
![2015-10-19_23 02 30](https://cloud.githubusercontent.com/assets/1819755/10598428/8969c4c2-76be-11e5-9851-a06f272b2e70.png)
Void Glass looks cool
![2015-10-19_23 03 06](https://cloud.githubusercontent.com/assets/1819755/10598431/91878ebe-76be-11e5-8d6b-b5e2b004c058.png)
Crafting Void Sticks.
![2015-10-19_23 04 00](https://cloud.githubusercontent.com/assets/1819755/10598447/b09a1c86-76be-11e5-819c-bd0cef07dcd4.png)
Crafting Void Tool
![2015-10-19_23 04 44](https://cloud.githubusercontent.com/assets/1819755/10598450/ba5a1956-76be-11e5-8767-e8447daca098.png)
Crafting Machine Core
![2015-10-19_23 09 43](https://cloud.githubusercontent.com/assets/1819755/10598453/c8b62120-76be-11e5-8897-a1e77c57495b.png)
The next section contains both elemental iron and elemental diamond. To get these
you'll need to create an elemental totem which hasn't been implemented yet. The next
update will have the totems completed hopefully
Crafting Fire Iron Block
![2015-10-19_23 12 47](https://cloud.githubusercontent.com/assets/1819755/10598495/2cf72864-76bf-11e5-8baf-74cbd9c4f658.png)
Crafting Water Iron Block
![2015-10-19_23 13 20](https://cloud.githubusercontent.com/assets/1819755/10598498/332e6b7a-76bf-11e5-9bab-6c3cba102f95.png)
Crafting Air Iron Block
![2015-10-19_23 13 26](https://cloud.githubusercontent.com/assets/1819755/10598505/3cf84734-76bf-11e5-8569-1ef07b1fbf34.png)
Crafting Earth Iron Block
![2015-10-19_23 13 32](https://cloud.githubusercontent.com/assets/1819755/10598516/4eb95698-76bf-11e5-898d-37c62c6ae738.png)
Crafting Fire Diamond Block
![2015-10-19_23 13 37](https://cloud.githubusercontent.com/assets/1819755/10598532/6129c7a4-76bf-11e5-8463-083a26170ab0.png)
Crafting Water Diamond Block
![2015-10-19_23 13 42](https://cloud.githubusercontent.com/assets/1819755/10598534/67d86b6e-76bf-11e5-8653-487e6caaad10.png)
Crafting Air Diamond Block
![2015-10-19_23 13 46](https://cloud.githubusercontent.com/assets/1819755/10598541/72cad084-76bf-11e5-970a-b5ca970ceae1.png)
Crafting Earth Diamond Block
![2015-10-19_23 13 54](https://cloud.githubusercontent.com/assets/1819755/10598559/a73f4174-76bf-11e5-9083-d2fe8c56e4ff.png)
And they all can be revered back to ingots and diamonds.
![2015-10-19_23 14 01](https://cloud.githubusercontent.com/assets/1819755/10598565/ba073df2-76bf-11e5-91e7-93866f7ef00b.png)
![2015-10-19_23 14 09](https://cloud.githubusercontent.com/assets/1819755/10598566/bb6a231c-76bf-11e5-94be-811eb1952d3b.png)
![2015-10-19_23 14 13](https://cloud.githubusercontent.com/assets/1819755/10598567/bc80a55a-76bf-11e5-8bcb-f6ecb87526e8.png)
![2015-10-19_23 14 16](https://cloud.githubusercontent.com/assets/1819755/10598568/bcf4474e-76bf-11e5-8bdb-612dd9ca85d8.png)
![2015-10-19_23 14 22](https://cloud.githubusercontent.com/assets/1819755/10598569/be63c6cc-76bf-11e5-89a2-045caf739239.png)
![2015-10-19_23 14 25](https://cloud.githubusercontent.com/assets/1819755/10598570/bf6e3b60-76bf-11e5-8fcd-20017a7638ec.png)
![2015-10-19_23 14 28](https://cloud.githubusercontent.com/assets/1819755/10598571/c087e3ac-76bf-11e5-9d2f-f06ac550da5f.png)
![2015-10-19_23 14 32](https://cloud.githubusercontent.com/assets/1819755/10598576/c3076e86-76bf-11e5-8f91-dd737ab03856.png)

Whew that was a lot of pictures. I'm going to keep this open for basic vanilla recipes for documentation purposes. I guess I'll need some sort of guide in the mod to help with starting
out. But for now I'll uses the git issues track as documentation. I'll setup the wiki later for 
documentation later well night.

**Comment by aod6060** [comment]
I've added in a new recipe for Void Dust Block. Its going to be used as a way to revert element iron and diamond blocks back to there original state. Because who know when you might be low on diamonds and iron :D. 

Void Dust Block Recipe
![void_dust_block_recipe](https://cloud.githubusercontent.com/assets/1819755/10654927/97b8c5e6-7835-11e5-8f48-81e23db8be71.png)

Also I've removed the elemental totem heads. I'll be using Elemental Dust Blocks as the top of the elemental totems for generating elemental iron and diamond blocks. I'll explain those today when I finish the void totem. I'll also add a new issue for totem recipes.
